icdc_schema.py

```diff
@@ -512,10 +513,15 @@ def _validate_type(self, model_type, str_value):
                     and not re.match(r'\bltf\b', str_value, re.IGNORECASE)):
                 return False, wrong_type
         elif model_type[PROP_TYPE] == 'Array':
-            for item in get_list_values(str_value):
-                validation_result, error_type = self._validate_type(model_type[ITEM_TYPE], item)
-                if not validation_result:
-                    return False, wrong_type
+            for item in get_list_values(str_value, self.delimiter):
+                if ENUM in model_type[ITEM_TYPE]:
```

Comment:
Will self._validate_type() work? Why rewrite it here?

Reply:
The code is part of the ._validate_type() function